src/Transport/AzureServiceBusSettings.cs

```diff
@@ -96,6 +96,19 @@ public AzureServiceBusSettings EntityMaximumSize(int maximumSizeInGB)
             return this;
         }
 
+        /// <summary>
+        /// Overrides the default time to auto delete queues and topics when idle.
+        /// </summary>
+        [ObsoleteEx(
```

Comment:
The setting (if added) shouldn't be obsoleted.

Reply:
removed setting from `AzureServiceBusSettings`